Jenkinsfile.public

```diff
@@ -63,6 +62,7 @@ pipeline {
                         sh 'go run scripts/use-ports.go -from 1024 -to 10000 &'
                         sh 'go test -parallel 4 -p 6 -vet=off $COVERFLAGS -timeout 20m -json -race ./... 2>&1 | tee .build/tests.json | go run ./scripts/xunit.go -out .build/tests.xml'
                         sh 'go run scripts/check-clean-directory.go'
+                        sh 'killall -3 use-ports'
```

Comment:
why kill this?

Reply:
it blocks ports `1024 - 10000` until end of the whole build and blocks emulator to start correctly.